middleware/index.js

```diff
@@ -0,0 +1,227 @@
+'use strict';
+var chalk = require('chalk');
+var yeoman = require('yeoman-generator');
+
+var wsModels = require('loopback-workspace').models;
+
+var actions = require('../lib/actions');
+var helpers = require('../lib/helpers');
+var validateName = helpers.validateName;
+var async = require('async');
+
+function listChoices(items) {
+  var choices = [];
+  items.forEach(function(item, index) {
+    choices.push({
+      name: (index + 1) + '. ' + item,
+      value: item
+    });
+  });
+  return choices;
+}
+
+module.exports = yeoman.generators.Base.extend({
+  // NOTE(bajtos)
```

Comment:
Please remove my name from the note as I am not the author of the note in this particular file. I think we can remove this line entirely and keep just the comment.